### PR TITLE
Commit for #535

### LINKED ISSUE #535: WebUI - Allow to change a bug title
Today when We select a bug It´s only possible to add comments but in the same page should be possible to change bug title too.

### PR DESCRIPTION
**Pull request by claudioantonio**
Fix #535

### CODE REVIEW

File: webui/src/components/BugTitleForm/BugTitleForm.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      return false;
+    }
+  }
+
+  function submitNewTitle() {
+    if (!isFormValid()) return;
+    setTitle({
+      variables: {
+        input: {
+          prefix: bug.humanId,
```
Comment (MichaelMure) [inline webui/src/components/BugTitleForm/BugTitleForm.tsx:90]: you are using `humanId` here as well. It works but can cause problem if you have multiple bugs with the same prefix. Better use `id`.

Comment (claudioantonio) [reply webui/src/components/BugTitleForm/BugTitleForm.tsx]: Fixed.
I also did a find in files and this was the only reference to humanId.

File: webui/src/components/BugTitleForm/SetTitle.graphql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+mutation setTitle($input: SetTitleInput!) {
+  setTitle(input: $input) {
+      bug {
+        humanId
```
Comment (MichaelMure) [inline webui/src/components/BugTitleForm/SetTitle.graphql:4]: this should be `id`, not `humanId`

**Review by MichaelMure — CHANGES_REQUESTED** [review]

**Comment by MichaelMure** [comment]
I merged #545 which include the same changes are here if I'm not mistaken, so this one is not necessary anymore.

Your chaining of branches/PRs make things quite confusing (and probably more difficult for you). Could you try to either always branch from master or make a single branch/PR?

**Comment by claudioantonio** [comment]
You are right @MichaelMure It turned things more complicated for me...
For sure. In the future I will follow Your suggestions.
Sorry for the inconvenience.

**Pull request closed without merging**